Xwt/Xwt.Backends/ICanvasCellViewFrontend.cs

```diff
@@ -33,7 +33,9 @@ public interface ICanvasCellViewFrontend: ICellViewFrontend
 		ApplicationContext ApplicationContext { get; }
 		void Draw (object ctxBackend, Rectangle cellArea);
 		Rectangle GetDrawingAreaForBounds (Rectangle cellBounds);
+		[Obsolete("Use OnGetRequiredSize (SizeConstraint widthConstraint)")]
```

Comment:
There is no `OnGetRequiredSize`

Reply:
Maybe also decrease verbosity by removing the parameter name.

Reply:
yep, thanks!